test/utils.py

```diff
@@ -336,7 +335,6 @@ def broadcast_subscriber(_comm, body, sender=None, subject=None, correlation_id=
 
     def test_broadcast_noone_listening(self):
         """ Check that it's not an error to send a broadcast if no one is listening"""
-        result = self.communicator.broadcast_send('into the ether')
-        self.assertIs(True, result)
+        self.communicator.broadcast_send('into the ether')
```

Comment:
I'm not sure why I removed this.  The return value doesn't really matter and I guess I was thinking of dropping it anyway because there's no possibility of `broadcast_send` returning anything other than `True`.  It can just except.  In any case given that the signature remains as is I will revert the check.

Reply:
Ok, I made a change I needed to in a the broadcast_send call because `aio_pika` wasn't returning true but a `Basic.Ack`.  The behaviour is now as before.